subsys/bluetooth/mesh/light_temp_srv.c

```diff
@@ -466,22 +475,25 @@ void bt_mesh_light_temp_srv_set(struct bt_mesh_light_temp_srv *srv,
 	(void)bt_mesh_lvl_srv_pub(&srv->lvl, NULL, &lvl_status);
 }
 
-enum bt_mesh_model_status
-bt_mesh_light_temp_srv_range_set(struct bt_mesh_light_temp_srv *srv,
-				 struct bt_mesh_msg_ctx *ctx,
-				 struct bt_mesh_light_temp_range *range)
+enum bt_mesh_model_status bt_mesh_light_temp_srv_range_set(struct bt_mesh_light_temp_srv *srv,
+							   struct bt_mesh_msg_ctx *ctx,
+							   struct bt_mesh_light_temp_range *range)
 {
 	const struct bt_mesh_light_temp_range old = srv->range;
 
-	if ((range->min < BT_MESH_LIGHT_TEMP_MIN) ||
-	    (range->min > range->max)) {
+	if (range->min < BT_MESH_LIGHT_TEMP_MIN && range->min != BT_MESH_LIGHT_TEMP_UNKNOWN) {
```

Comment:
I can't seem to get this to make sense ... wouldn't `range->min < TEMP_MIN` mean that `range->min != TEMP_UNKNOWN` is always true?